main.py

```diff
@@ -67,23 +68,22 @@ def main(is_simulation: bool, real_servo: bool) -> None:
             # Update the airbrakes finite state machine
             airbrakes.update()
 
-            if is_simulation:
-                # This is what prints the flight data to the console in real time, we only do it when running the sim
-                # because printing a lot of things can significantly slow down the program
-                flight_display.update_display()
+            if args.mock:
+                if not args.debug:  # Don't print the flight data if we are in debug mode
+                    # This is what prints the flight data to the console in real time, we only do
+                    # it when running the sim because printing a lot of things can significantly slow down the program
+                    flight_display.update_display()
                 # Stop the sim when the data is exhausted:
                 if not airbrakes.imu._data_fetch_process.is_alive():
                     flight_display.update_display(end_sim=FlightDisplay.NATURAL_END)
                     break
     except KeyboardInterrupt:
-        if is_simulation:
+        if args.mock:
             flight_display.update_display(end_sim=FlightDisplay.INTERRUPTED_END)
     finally:
         airbrakes.stop()
 
 
 if __name__ == "__main__":
-    # We check if the user has passed the MOCK_ARGUMENT (m) or REAL_SERVO_ARGUMENT (r) to the program
-    # If you want to run a simulation on your computer (or Pi) run: python main.py m
-    # If you want to run a simulation on the Pi with the servo connected run: python main.py m rs
-    main(len(sys.argv) > 1 and MOCK_ARGUMENT in sys.argv[1:], len(sys.argv) > 1 and REAL_SERVO_ARGUMENT in sys.argv[1:])
+    args = arg_parser()  # Load all command line options
```

Comment:
I would add a comment here briefly describing each argument and what it does--it just makes it a little easier for someone new going through the code to immediately understand things